Tasks/SonarQubePostTest/PRCA/PostComments-Module.psm1

```diff
@@ -50,289 +52,195 @@ $script:project = $null
 $script:pullRequest = $null
 $script:artifactUri = $null 
 
+$script:messageSource = $null
```

Comment:
I now keep the message source and the resolved comment message as members of the module as I realized I was passing them around on every function.